template/{{.input_root_dir}}/.github/workflows/{{.input_project_name}}-run-tests.yml.tmpl

```diff
@@ -2,8 +2,6 @@ name: {{ if (eq .input_include_feature_store `yes`) }}Feature and {{end}}Trainin
 on:
   workflow_dispatch:
   pull_request:
-    paths:
```

Comment:
The idea was to also run the tests when the `run-tests.yml` file was also updated in case the user updates it. I added the path and also added a second path to trigger the run when the yml file is updated.

Reply:
Can you add this for ADO workflow as well

Reply:
Bump